.github/workflows/backup_audit_logs.yml

```
@@ -17,11 +17,19 @@ jobs:
       - name: Install dependencies
         run: npm i
 
-      - name: Fetch audit logs
-        run: node bot.mjs
+      - name: Linter
+        run: npm run lint
+
+      - name: Prettier
+        run: npm run format
+
+      - name: Compile TypeScript
+        run: npm run dev
```

```suggestion
      - name: Compile TypeScript
        run: npm run build
```
This won't do what you think, as it'll watch the files and fail to exit. Build instead.